src/consensus/state.ml

```diff
@@ -0,0 +1,61 @@
+open Crypto
+open Protocol
+module Operation_map = Map.Make (Operation)
+
+type identity = {
+  secret : Secret.t;
+  key : Key.t;
+  t : Key_hash.t;
+  uri : Uri.t;
+}
+
+type state = {
```

Comment:
why not type t?

Reply:
because of the name

Reply:
so that open will not show `t -> Block.t -> t` everywhere